code/game/atoms.dm

```diff
@@ -1230,6 +1230,8 @@
 			log_game(log_text)
 		if(LOG_MECHA)
 			log_mecha(log_text)
+		if(LOG_RADIO_EMOTE)
+			log_radio_emote(log_text)
```

Comment:
is this if in switch? It doesn't look like it is on my side but it might as well be github

Reply:
yes,